src/components/Tailwind/InputFields/SlippageTolerance.tsx

```diff
@@ -30,14 +32,22 @@ interface SlippageToleranceProps {
   didChangeValue: (val: string) => void
 }
 
+const TooltipContainer = styled.div`
+  cursor: pointer;
+`
+
 const SlippageTolerance = ({ value, didChangeValue }: SlippageToleranceProps) => {
   const [customSlippage, setCustomSlippage] = useState('')
-
+  const Description = 'Your transaction will revert if the price changes unfavorably by more than this percentage.'
```

Comment:
Pls use translation, e.g. 
```suggestion
  const Description = t('Your transaction will revert if the price changes unfavorably by more than this percentage.')
```